src/NativeApi/PushNotificationIOS.elm

```diff
@@ -0,0 +1,14 @@
+module NativeApi.PushNotificationIOS exposing (register)
+
+{-|
+@docs register
+-}
+
+import Native.NativeUi.PushNotificationIOS
+import Task exposing (Task)
+
+
+{-| -}
+register : Task String String
+register =
+    Native.NativeUi.PushNotificationIOS.register "ignored"
```

Comment:
I don't know why, but if I don't pass an argument to this JS function, I get an error/crash. Passing a string value that I ignore in the JS makes this not error/crash. Any thoughts on why that's the case?

Reply:
@eeue56 might have an idea?

Reply:
I'll take a look later today

Reply:
@eeue56 Did you have time to look at this?

Reply:
looking now, sorry

Reply:
There is no such thing as an arg-less function from Elm. Something without an arg is a value. So a `Task String String` is a value. You'd need to change the code to this: 

```
const register = _elm_lang$core$Native_Scheduler.nativeBinding(function(callback) {
    PushNotificationIOS.addEventListener('register', token => {
      callback(_elm_lang$core$Native_Scheduler.succeed(token));
    });
    PushNotificationIOS.addEventListener('registrationError', e => {
      callback(_elm_lang$core$Native_Scheduler.fail(e.message));
    });
    PushNotificationIOS.requestPermissions();
  });

```

Because then you've constructed the value, not returned a function

Reply:
Thanks for the explanation! I'll update this accordingly